graylog2-server/src/main/java/org/graylog2/security/realm/LdapUserAuthenticator.java

```diff
@@ -250,7 +249,8 @@ private void updateFromLdap(User user, LdapEntry userEntry, LdapSettings ldapSet
             translatedRoleIds.addAll(user.getRoleIds());
         }
         user.setRoleIds(translatedRoleIds);
-        user.setPermissions(Collections.emptyList());
+        // preserve the raw permissions (the ones without the synthetic self-edit permissions or the "*" admin one)
+        user.setPermissions(user.getPermissions());
```

Comment:
![I can has unit test for this?](https://cloud.githubusercontent.com/assets/43951/17108018/fa285368-5291-11e6-8b10-1627b177f6c3.jpg)